server/runtime/src/main/java/org/infinispan/server/configuration/ServerConfigurationParser.java

```diff
@@ -1612,4 +1613,24 @@ private static void parseConnectorIpFilter(ConfigurationReader reader, IpFilterC
          }
       }
    }
+
+   @Override
+   public void readAttribute(ConfigurationReader reader, String elementName, int attributeIndex, ConfigurationBuilderHolder holder) {
+      if (org.infinispan.configuration.parsing.Element.forName(elementName) == org.infinispan.configuration.parsing.Element.TRANSPORT) {
+         ServerConfigurationBuilder serverBuilder = holder.getGlobalConfigurationBuilder().addModule(ServerConfigurationBuilder.class);
+         String attributeName = reader.getAttributeName(attributeIndex);
+         switch (Attribute.forName(attributeName)) {
+            case SECURITY_REALM:
+               serverBuilder.security().transport().securityRealm(reader.getAttributeValue(attributeIndex));
+               break;
+            case SNI_DOMAINS:
```

Comment:
I don't need the SNI_DOMAINS anymore. If you want, you could remove the attribute and all the related code.

Reply:
Removed